MAUI/DataForm/migration.md

```diff
@@ -174,6 +174,18 @@ this.Content = dataForm;
 <td>Represents the class that encapsulates the layout and text editor setting for a data field.</td>
 </tr>
 
+<tr>
+<td>{{'[DataFormNumericItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormNumericItem.html)'| markdownify }}</td>
+<td>DataFormNumericItem</td>
+<td>Represents the class that encapsulates the layout and numeric editor(SfNumericEntry) settings for data field.</td>
+</tr>
+
+<tr>
+<td>{{'[DataFormMaskedEditTextItem](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html)'| markdownify }}</td>
+<td>DataFormMaskedTextItem</td>
+<td>Represents the class that encapsulates the layout and masked text editor(SfMaskedEntry) settings for data field.</td>
```

Comment:
<td>Represent the class that encapsulates the layout and masked text editor(SfMaskedEntry) settings for data field.</td>

Reply:
Content changed